File: seleniumTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;

namespace SeleniunTest
{
    class Program
    {
        static void Main(string[] args)
        {
            IWebDriver driver = new ChromeDriver();

            driver.Navigate().GoToUrl("https://testing.todorvachev.com");

            Thread.Sleep(3000);

            driver.Quit();

        }
    }
}

File: Java/SeleniumTest.java
package com.selenium;

import net.bytebuddy.implementation.bind.MethodDelegationBinder;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;


public class Main {

    public static void main(String[] args) throws InterruptedException {
	
        WebDriver driver = new ChromeDriver();

        driver.get("https://testing.todorvachev.com");
        Thread.sleep(15000);
        System.out.printf("test");
        driver.quit();
    }
}